extensions/resteasy-reactive/quarkus-resteasy-reactive-links/deployment/src/main/java/io/quarkus/resteasy/reactive/links/deployment/LinksProcessor.java

```diff
@@ -188,6 +188,8 @@ private RuntimeValue<GetterAccessorsContainer> implementPathParameterValueGetter
                                 entityType, getterMetadata.getFieldName(), getterMetadata.getGetterAccessorName());
                     }
                 }
+
+                getterAccessorsContainerRecorder.validateContainer(getterAccessorsContainer, entityType);
```

Comment:
Wouldn't it be easier to move this one line below and validate all the entities at once? Also you could list all the entities without ids in one go in the exception you throw? Otherwise it's going to be a bit tedious when you have several entities without the id.

All in all, I think it should also simplify your code (unless I'm missing something).

Reply:
I was keeping track of the `entityType`
I pushed a modification of the validation to check all the container only once and fixed the test